examples/specs_integration/src/sync/entity_move.rs

```diff
@@ -0,0 +1,46 @@
+use godot::*;
+use specs::prelude::*;
+
+use std::collections::HashMap;
+
+use euclid::{vec3, Angle, Transform3D};
+use specs::world::Index;
+use sync::Pos;
+
+pub struct GdEntityTransform<'a> {
+    pub spatials: &'a mut HashMap<Index, Spatial>,
+}
+
+/// InShipSpace is just used to make euler happy and help us track units...
+pub struct InShipSpace;

```

Comment:
There are no other units in this example. Seems like we can just use `UnknownUnit` here?